File: excelium-model/src/main/java/excelium/model/test/item/Item.java
/*
 * MIT License
 *
 * Copyright (c) 2018 Excelium
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package excelium.model.test.item;

/**
 * Represents item model.
 *
 * @author PhungDucKien
 * @since 2018.03.29
 */
public class Item {

    /** Name */
    private String name;

    /** Description */
    private String description;

    /** Value */
    private String value;

    /** Android value */
    private String androidValue;

    /** iOS value */
    private String iosValue;

    /**
     * Gets name.
     *
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * Sets name.
     *
     * @param name the name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * Gets description.
     *
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * Sets description.
     *
     * @param description the description
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * Gets value.
     *
     * @return the value
     */
    public String getValue() {
        return value;
    }

    /**
     * Sets value.
     *
     * @param value the value
     */
    public void setValue(String value) {
        this.value = value;
    }

    /**
     * Gets android value.
     *
     * @return the android value
     */
    public String getAndroidValue() {
        return androidValue;
    }

    /**
     * Sets android value.
     *
     * @param androidValue the android value
     */
    public void setAndroidValue(String androidValue) {
        this.androidValue = androidValue;
    }

    /**
     * Gets ios value.
     *
     * @return the ios value
     */
    public String getIosValue() {
        return iosValue;
    }

    /**
     * Sets ios value.
     *
     * @param iosValue the ios value
     */
    public void setIosValue(String iosValue) {
        this.iosValue = iosValue;
    }
}

File: packages/model/src/test/item/Item.ts
/*
 * Licensed to the Software Freedom Conservancy (SFC) under one
 * or more contributor license agreements.  See the NOTICE file
 * distributed with this work for additional information
 * regarding copyright ownership.  The SFC licenses this file
 * to you under the Apache License, Version 2.0 (the
 * "License"): string; you may not use this file except in compliance
 * with the License.  You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */

/**
 * Represents item model.
 *
 * @author PhungDucKien
 * @since 2018.03.29
 */
export default class Item {
  /**
   * Name
   */
  public name: string;

  /**
   * Description
   */
  public description: string | null;

  /**
   * Value
   */
  public value: string | null;

  /**
   * Android value
   */
  public androidValue: string | null;

  /**
   * iOS value
   */
  public iosValue: string | null;
}
